skyportal/handlers/api/source.py

```diff
@@ -394,7 +394,7 @@ def post(self):
         DBSession().commit()
 
         self.push_all(action="skyportal/FETCH_SOURCES")
-        self.push_all(action="skyportal/FETCH_CANDIDATES")
+        self.push_all(action="skyportal/REFRESH_CANDIDATE", payload={"id": obj.id})
```

Comment:
This should also use the internal_key so as to not leak sensitive obj IDs

Reply:
fixed